frontend/gpay_agent/lib/agent_views/loading_merchant.dart

```diff
@@ -3,13 +3,15 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:sleek_circular_slider/sleek_circular_slider.dart';
 
+
```

Comment:
Please remove these newline. Using the formatter should help you with this

Reply:
this is part of the agent app. Pragya has worked on that not me.